docs/knn/approximate-knn.md

```diff
@@ -157,3 +157,5 @@ A space corresponds to the function used to measure the distance between 2 point
     <td>1 / (1 + Distance Function)</td>
   </tr>
 </table>
+
+*Note* - the cosine similarity formula does not include the `1 - ` prefix. However, because nmslib equates smaller scores to closer results, they return `1 - cosineSimilarity` for their cosine similarity space; that is why the `1 - ` is included in the distance function.
```

Comment:
IMO, `1 - ` is not a prefix, can we say why we subtract from 1.0 ? what do you think?